turbo.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://turbo.build/schema.json",
+  "$schema": "https://turbo.build/schema.v1.json",
```

Comment:
we were pointing to "latest" version and with turborepo v2 schema was showing errors (pipeline prop is not in v2)